src/getCoinCombination.test.js

```diff
@@ -7,4 +7,28 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should be 1 penny if "cents" = 1', () => {
+    const result = getCoinCombination(1);
+
+    expect(result).toStrictEqual([1, 0, 0, 0]);
+  });
+
+  it('should be 1 penny + 1 nickel if "cents" = 6', () => {
+    const result = getCoinCombination(6);
+
+    expect(result).toStrictEqual([1, 1, 0, 0]);
+  });
+
+  it('should be 2 pennies + 1 nickel + 1 dime if "cents" = 17', () => {
+    const result = getCoinCombination(17);
+
+    expect(result).toStrictEqual([2, 1, 1, 0]);
+  });
+
+  it('should be 2 2 quarters if "cents" = 50', () => {
```

Comment:
There is a typo in the test description. It should be '2 quarters' instead of '2 2 quarters'.